src/InstrumentationEngine/CorMethodMalloc.cpp

```diff
@@ -13,6 +13,8 @@ MicrosoftInstrumentationEngine::CCorMethodMalloc::CCorMethodMalloc() : m_cbBuffe
 
 PVOID MicrosoftInstrumentationEngine::CCorMethodMalloc::Alloc(_In_ ULONG cb)
 {
+    //TODO Any consecutive calls to Alloc will destroy the previous buffer.
```

Comment:
Consider failing subsequent calls or leaking the buffer

Reply:
leak + log

Reply:
It looks like this is the default code flow; we don't have independent cleanup for this buffer, we rely on subsequent AllocCalls or ref counting to cleanup. I will leave this as is for now.